controllers/status/status.go

```diff
@@ -210,7 +212,93 @@ func RemoveComponentCondition(conditions *[]conditionsv1.Condition, component st
 	conditionsv1.RemoveStatusCondition(conditions, conditionsv1.ConditionType(component+ReadySuffix))
 }
 
-// ModelRegistryStatus struct holds the status for the ModelRegistry component.
+type ComponentObject interface {
```

Comment:
Is it used anywhere? grep does not show any place.

Reply:
removed